src/components/ModalLedgerSignToken.js

```diff
@@ -119,10 +119,10 @@ function ModalLedgerSignToken({token, modalId, cb, onClose}) {
     } else {
       switch (arg.error.status) {
         // user deny
-        case 0x6985: 
+        case 0x6985:
           setError('user_deny');
           break;
-        case 0xb00a: 
+        case 0xb00a:
```

Comment:
I know you just removed the empty space, but should these be constants?

Reply:
Created an enum with all known error codes